README.md

```diff
@@ -36,3 +36,12 @@ Here's an excellent guide: [How to execute robot framework tests](https://gist.g
 ## License ##
 
 This code is released under the MIT license.
+
+## Istalling Pycharm ##
```

Comment:
Move to row 30ish. and make level 3 (`###`). That way it's organized under the heading "How to run tests from IDE"